src/libsync/graphapi/spacesmanager.cpp

```diff
@@ -43,6 +43,8 @@ SpacesManager::SpacesManager(Account *parent)
 
     connect(_refreshTimer, &QTimer::timeout, this, &SpacesManager::refresh);
     connect(_account, &Account::credentialsFetched, this, &SpacesManager::refresh);
+    // legacy signal which is going to be removed in 5.0
```

Comment:
```suggestion
    // TODO: legacy signal which is going to be removed in 5.0
```

Makes this a little more visible in common IDEs.

Reply:
I have already a pr for that :D